sphinx_gallery/gen_rst.py

```diff
@@ -391,6 +391,23 @@ def handle_exception(exc_info, src_file, script_vars, gallery_conf):
         stack = stack[2:]
     else:
         stack = stack[1:]
+    start = 0
```

Comment:
I would add here a big-picture comment on the purpose of this code block.

Reply:
Done